modules/govuk/manifests/nodes/postgresql_db_admin.pp

```diff
@@ -64,16 +69,21 @@
   # We don't actually want to run a local PostgreSQL server, just
   # configure the RDS one
   class { '::postgresql::server':
+    package_ensure           => $ensure,
     default_connect_settings => $default_connect_settings,
     service_manage           => false,
   } ->
 
   service { 'postgresql':
-    ensure  => stopped,
+    ensure => stopped,
   }
 
-  include ::govuk_postgresql::server::not_slave
+  if $ensure == present {
+    include ::govuk_postgresql::server::not_slave
```

Comment:
I think you'll want the `service { 'postgresql'` block in this conditional too. I don't know how that will behave if the service is removed.

Reply:
Good shout!